docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,128 @@
+
+
+********
+Overview
+********
+Once downloaded, extract and run the Vertiq Testing Tool executable. When running this application for the first time, 
+you may encounter a Windows Defender SmartScreen warning.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender.png
+
+Click on 'More info'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender_more_info.png
+
+Then click 'Run anyway'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/welcome_screen.png
+
+Once the application loads, you will be presented with the welcome screen. Follow the instructions on this screen to connect your module.
+
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input.png
+
+Once connected, you should see the tab as shown above.
+
+#. The :gold:`Tabs` section provides different widgets for controlling your module. ESC Input provides configuration parameters necessary for integrating properly with a flight controller as well as simulating flight controller throttle commands via the IQUART raw value message.
+
+#. The :green:`Plotting` section contains a live plot in the upper section and a dashboard with widgets to control the live plot in the lower section.
+
+#. The :purple:`Control Panel` section contains widgets to configure specific parameters and to control your module. The Control Panel on each tab contains different widgets.
+
+In the ESC Input tab shown above, the Control Panel contains the same parameters necessary to configure your module for use with a flight controller as in this :ref:`Control Center example <flight_controller_config_with_control_center>`.
+
+
+==================
+Continuous command
+==================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input_command_widget.png
+
+A notable feature in the ESC Input tab is the 'Continuous command' toggle at the bottom of the Control Panel. This feature is disabled by default, which means you need to click the 'Command ESC' button every time you want to send an ESC command.
+When this feature is enabled, the ESC command will be continuously sent at the 'Command rate' with 10 Hz being the default value. The 'Command rate' can be adjusted to send the ESC command at different rates. 
+You can also use the slider to adjust the ESC input value, which mimics adjusting the flight controller's throttle input between [0, 1].
+
+
+================================
+Live Plotting and Active Metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. 
+Please refer to the :ref:`IQUART Client Table Reference tables <iquart_client_reference_tables>` to find the endpoints that allows for 'get' in the Access field, 
+as these are the supported metrics that can be plotted.
```

Comment:
people will also have to check their module family page to see what clients are supported